server/models/Reviews.js

```diff
@@ -0,0 +1,31 @@
+const { Schema, model } = require('mongoose');
+
+const ReviewsSchema = new Schema({
+  userId: {
+    type: Schema.Types.ObjectId,
+    ref: 'User',
+  },
+  reviews: [
+    {
+      reviewerId: {
+        type: Schema.Types.ObjectId,
+        ref: 'User',
+      },
+      rating: {
+        type: Number,
+        min: 1,
+        max: 5,
+        required: true,
+      },
+      createdAt: {
```

Comment:
You can use mongoose timestamps, it creates these fields for you automatically :)